mmv1/products/lustre/Instance.yaml

```diff
@@ -158,3 +158,52 @@ properties:
     description: |-
         The reason why the instance is in a certain state.
     output: true
+  - name: accessRulesOptions
+    type: NestedObject
+    description: |-
+      Access control rules for the Lustre instance. Configures default root
+      squashing behavior and specific access rules based on IP addresses.
+    properties:
+      - name: defaultSquashMode
```

Comment:
Looks like this is required according to the [API doc](https://docs.cloud.google.com/managed-lustre/docs/reference/rest/v1/projects.locations.instances#accessrulesoptions)